setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='tin-score-calculation',
-    version='0.4',
+    version='0.4.1',
```

Comment:
If the method is calculating something different now (regardless if it was correct then or now) I think we should bump up major ver. (?)

Reply:
Ok, 0.5 then?

Reply:
OK `0.5.0` is fine for me, since we don't have `1.0.0` yet. This would be a minor version bump then.

Reply:
OK, done.